build/test-replacements.js

```diff
@@ -214,3 +231,12 @@ module.exports['test-stream-pipe-cleanup-pause.js'] = [
     'var '
   ]
 ]
+module.exports['test-stream2-readable-empty-buffer-no-eof.js'] = [[
+  /let /g,
+  'var '],
+  [
+    `var buf = Buffer(5).fill('x');`,
```

Comment:
8 keystrokes were saved in the commit that added this I assume because it looked cool